packages/button/src/LionButton.js

```diff
@@ -35,7 +35,13 @@ export class LionButton extends DisabledWithTabIndexMixin(LitElement) {
   }
 
   render() {
-    return html` <div class="button-content" id="${this._buttonId}"><slot></slot></div> `;
+    return html`
+      <div class="button-content" id="${this._buttonId}">
+        <slot name="prefix"></slot>
+        <slot></slot>
```

Comment:
Sorry for not responding for so long, we had a button story on our backlog, which made sense to wait upon since it touched this topic. But the story got postponed a few times.

How about making use of a template, which can easily be overridden to your taste?

```js
_buttonContentTemplate() {
  return html`<slot></slot>`;
}
    
render() {
    return html` <div class="button-content" id="${this._buttonId}">this._buttonContentTemplate()</div> `;
```